app/helpers/users_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UsersHelper
+  def available_times_of_day
+    (0...24).map { |hour| Time.utc(2000, 1, 1, hour, 00, 0) }
```

Comment:
Thanks @MonikaMahanthappa, I agree.  I wanted to create the simplest PR possible so I left out user time zones. 

If we present the times as UTC and save the times as UTC we have a working feature and a solid base to build user time zones on.

Do you think this approach is ok?